crafters/image/ImageReader/__init__.py

```diff
@@ -39,4 +39,23 @@ def craft(self, buffer: bytes, uri: str, *args, **kwargs) -> Dict:
         img = np.array(raw_img).astype('float32')
         if self.channel_axis != -1:
             img = np.moveaxis(img, -1, self.channel_axis)
-        return dict(weight=1., blob=img)
+        # Add image metadata
+        from PIL.ExifTags import TAGS
```

Comment:
while I really like the feature, I think it would be good to have this activated with an instance `argument`.